### DISCUSSION TITLE
Set values of query params at design time for preview purposes.

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7479] [Ideas]
The scenario : you are designing you complex screen for editing your lets say "Projects" on the /projects/:id route.
Now, the route is previewed with a random? record from project, but you want to design based on a specific row in preview, because that specific row you pick, triggers all the conditionals you want, or has related/child records you want to see in preview to assist you in designing. 

under the hood, this is simple setting the URL.id to the id of the selected row, and your design preview will show that one.

The proposed solution : let the user set preview values of query params at design time, as properties of the screen.

Just and idea!

**Comment by melohagan** [comment]
Great idea - there was another similar idea recently too!

https://github.com/Budibase/budibase/issues/7502
